src/fu-util-common.c

```diff
@@ -909,7 +909,7 @@ fu_util_convert_description (const gchar *xml, GError **error)
 	return fu_common_strstrip (str->str);
 }
 
-static gchar *
+gchar *
```

Comment:
Maybe adding gtk-doc here might be a good idea. I assume @tmp is in seconds?

Reply:
OK, fixed up.